convert2rhel/unit_tests/actions/pre_ponr_changes/handle_packages_test.py

```diff
@@ -255,7 +271,7 @@ def test_remove_repository_files_packages_not_removed(
 
 
 def test_remove_repository_files_packages_dependency_order(remove_repository_files_packages_instance):
-    expected_dependencies = ("BACKUP_REDHAT_RELEASE",)
+    expected_dependencies = ("BACKUP_REDHAT_RELEASE", "BACKUP_REPOSITORY")
```

Comment:
There is failing test, I think adding one more item is enough to get it working since @bocekm added this item in the last commit
```suggestion
    expected_dependencies = ("BACKUP_REDHAT_RELEASE", "BACKUP_REPOSITORY", "PRE_SUBSCRIPTION")
```

Reply:
Right, this unit test failure follows my last push. Fixing..